uamodeler/uamodeler.py

```diff
@@ -324,8 +324,9 @@ def really_exit(self):
 
         return True
 
-    def _after_add(self, new_node):
-        self._new_nodes.append(new_node)
+    def _after_add(self, new_nodes):
+        for new_node in new_nodes:
+            self._new_nodes.append(new_node)
```

Comment:
maybe use _new_nodes.extend(new_nodes). It is more efficient. (fix it in next PR)
Otherwise the PR looks all good. I can merge if you want